openshift/system/config/core.yml

```diff
@@ -1,5 +1,5 @@
 base: &default
-  url: <%= ENV.fetch('THREESCALE_CORE_INTERNAL_API', 'http://backend-listener:3000/internal/') %>
+  url: <%= ENV.fetch('BACKEND_ROUTE', 'http://backend-listener:3000/internal/') %>
```

Comment:
Why do we need to define this twice?

https://github.com/3scale/porta/blob/7fabcc4ef1b25e498f76511f2a0b42c04170bdf2/openshift/system/config/backend.yml#L2-L3

Reply:
because this config is being used in here: https://github.com/3scale/porta/blob/master/config/initializers/core.rb#L6

Reply:
That doesn't tell me why we need it twice 🙈 if it is the same thing, we should define it only once and then from the code point to the same place, no? 😕 🤔